selenium_tests/base.py

```diff
@@ -1,4 +1,9 @@
-"""Basic selenium tests for MicroMasters"""
+"""
+NOTE: THIS MODULE IS DEPRECATED AND SHOULD BE REMOVED ONCE THE 'snapshot_dashboard_states' COMMAND IS
```

Comment:
Yes I'm fine with the scope increase. It's a little complicated but hopefully fairly localized to that file